tests/dialects/test_bigquery.py

```diff
@@ -251,6 +251,7 @@ def test_bigquery(self):
                 "duckdb": "SELECT STRFTIME(CAST('2023-12-25' AS DATE), '%Y%m%d')",
             },
         )
+        self.validate_all("SELECT COUNTIF(x)", write={"duckdb": "SELECT COUNT_IF(x)"})
```

Comment:
Let's also test the BQ -> BQ and DuckDB -> BQ cases

Reply:
Ah @tobymao beat me to it, lol

Reply:
oh, I hadn't thought about the duckdb -> bigquery case.

it actually fails, I am not sure why though. I think it now thinks duckdb should not have an underscore.

```
AssertionError: 'SELECT COUNT_IF(x)' != 'SELECT COUNTIF(x)'
- SELECT COUNT_IF(x)
?             -
+ SELECT COUNTIF(x)
```

Reply:
thanks for the quick iteration, you're missing a rename func in the generator

Reply:
oh, the rename_func I believe